src/Microsoft.VisualStudio.Editors/Microsoft.VisualStudio.Editors.vbproj

```diff
@@ -50,7 +50,7 @@
       <SubType>Designer</SubType>
     </EmbeddedResource>
     <EmbeddedResource Update="OptionPages\GeneralOptionPageResources.resx">
-      <Generator>ResXFileCodeGenerator</Generator>
+      <Generator>PublicResXFileCodeGenerator</Generator>
```

Comment:
Can we add a comment here explaining why this needs to be public so we don't break this again?